legal-api/report-templates/template-parts/registration/completingParty.html

```diff
@@ -12,11 +12,7 @@
                         {% if party.officer.partyType == 'organization' %}
                            <span class="capitalize-text">{{ party.officer.organizationName }}</span>
                         {% else %}
-                           <span class="capitalize-text">{{ party.officer.lastName }}</span>,
-                           <span class="capitalize-text">{{ party.officer.firstName }}</span>
-                           {% if party.officer.middleName %}
-                              <span class="capitalize-text">{{ party.officer.middleName }}</span>
-                           {% endif %}
+                           <span class="capitalize-text">{{ party.officer.fullName }}</span>
```

Comment:
update `party.officer.fullName` to be `party.officer.name`.  

We can also remove `{% if party.officer.partyType == 'organization' %}` and `{% else %}` blocks and just use `<span class="capitalize-text">{{ party.officer.name }}</span>` for both scenarios.